app/models/conversion_host.rb

```diff
@@ -3,11 +3,18 @@ class ConversionHost < ApplicationRecord
 
   acts_as_miq_taggable
 
+  VALID_RESOURCE_TYPES = %w(Vm Host).freeze
+
   belongs_to :resource, :polymorphic => true
   has_many :service_template_transformation_plan_tasks, :dependent => :nullify
   has_many :active_tasks, -> { where(:state => 'active') }, :class_name => ServiceTemplateTransformationPlanTask, :inverse_of => :conversion_host
   delegate :ext_management_system, :hostname, :ems_ref, :to => :resource, :allow_nil => true
 
+  validates :name, :presence => true
+  validates :resource_type, :presence => true, :inclusion => { :in => VALID_RESOURCE_TYPES }
```

Comment:
Consider to use : `belongs_to :resource, :polymorphic => true` for `resource_type/id`

Reply:
@jameswnl I was very confused, lol.